modules/demo-magento/cypress/integration/checkout.js

```diff
@@ -57,4 +62,20 @@ describe('Checkout', () => {
 
     checkThankYouPageVisibility()
   })
+
+  it('add valid coupon', () => {
+    getAddCouponButton().click()
+    getCouponCodeInput().type('coupon_cypress_test')
+    getApplyCouponButton()
```

Comment:
Why does getApplyCouponButton also clicks the button? Either rename it to clickApplyCouponButton or add .click() method here as with getAddCouponButton